ui/load/load.go

```diff
@@ -59,6 +59,8 @@ type Icons struct {
 	TicketExpiredIcon,
 	TicketRevokedIcon,
 	TicketUnminedIcon *decredmaterial.Image
+
+	DexIcon, DexIconInactive, BTC, DCR, LTC, BCH *decredmaterial.Image
```

Comment:
What's the plan for the LTC and BCH icons? Is there any plan to use them currently? I don't see it, we can remove these icons and add them later if and when they become useful.